### PR TITLE
NR-101029: Add automatic releases

### PR DESCRIPTION
**Pull request by marcsanmi**
This PR adds Automatic Releases by leveraging the `release-toolkit` and [this](https://github.com/newrelic/newrelic-coreint-automation/blob/master/.github/workflows/trigger_prerelease.yaml) shared workflow.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
```
Comment (paologallinaharbur) [inline CHANGELOG.md:8]: can we add a link to the rt repo so that we know what we need to change?

File: .github/workflows/trigger_prerelease.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * 2"
```
Comment (paologallinaharbur) [inline .github/workflows/trigger_prerelease.yml:10]: can we add a comment saying ina a human-readable way when it is?

File: .github/workflows/trigger_prerelease.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+    secrets:
+      bot_token: "${{ secrets.COREINT_BOT_TOKEN }}"
```
Comment (paologallinaharbur) [inline .github/workflows/trigger_prerelease.yml:16]: Just adding a comment as a reminder to add secrets :)

Comment (marcsanmi) [reply .github/workflows/trigger_prerelease.yml:16]: Thanks for the reminder, already added by David yesterday 👍

File: CHANGELOG.md (hunk 1 above)
Comment (marcsanmi) [reply CHANGELOG.md:10]: Done in 16b45b0bce974a491205387f7b7e57c1f5d15947.

File: .github/workflows/trigger_prerelease.yml (hunk 2 above)
Comment (marcsanmi) [reply .github/workflows/trigger_prerelease.yml:11]: Done in 16b45b0bce974a491205387f7b7e57c1f5d15947.

File: .github/workflows/trigger_prerelease.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
```
Comment (gsanchezgavier) [inline .github/workflows/trigger_prerelease.yml:5]:
```suggestion
# For more details about how to release follow https://github.com/newrelic/coreint-automation/blob/main/docs/release_runbook.md
```

**Review by gsanchezgavier — APPROVED** [review]
just a nit thanks!

**Pull request merged**